lib/Zonemaster/Backend/Validator.pm

```diff
@@ -96,6 +98,11 @@ Readonly my $RELAXED_DOMAIN_NAME_RE => qr/^[.]$|^.{2,254}$/;
 Readonly my $TEST_ID_RE             => qr/^[0-9a-f]{16}$/;
 Readonly my $USERNAME_RE            => qr/^[a-z0-9-.@]{1,50}$/i;
 
+# Boolean
+Readonly my $BOOL_TRUE_RE           => qr/^(true|yes|1)$/i;
+Readonly my $BOOL_FALSE_RE          => qr/^(false|no|0)$/i;
```

Comment:
Would you consider removing the 1 and 0? I think it's a mistake to have both integers and booleans together using numeric representations.

Reply:
Done